packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -72,13 +72,15 @@ class AnalyzeContinuously extends AnalyzeBase {
   void _handleAnalysisStatus(AnalysisServer server, bool isAnalyzing) {
     if (isAnalyzing) {
       analysisStatus?.cancel();
+      analysisStatus = null;
```

Comment:
This is redundant since you set it in both cases below.

Reply:
ok, removed that line. I had added it just for consistency.